README.md

```diff
@@ -11,3 +17,28 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+Looking at the relationship between multiple inputs into both $\log_{2} n$ and $\log_{5} n$
+| Input for n              | $\log_{2} n$  |    $\log_{5} n$      |
+|----------------------------|:-----------:|------------------------|
+| 1                  |      0      |     0    |
+| 2                  |      2      |     0.431    |
+| 3                  |       1.585     |     0.683    |
+| 4                  |      2      |     0.861    |
+| 5                  |      2.322      |      1   |
+| 6                  |      2.585      |     1.113    |
+| 7                  |      2.807      |      1.209    |
+
+By analyzing the change of base rule of log, I noticed that ultimately all forms of log can be converted into $\log_{a} n = \frac{\log_{x} n}{\log_{x} a}$ where x can be an arbitrary or chosen input. 
+From this, we can determine that the base of a log is constantly scaling and so therefore can be disregarded just as constants are disregarded in analysis, as they have very little impact on the asymptote.
+By comparing the difference in values between $\log_{2} n$ and $\log_{5} n$ in this case equivalent to $\log_{2} n = \frac{\log_{x} n}{\log_{x} 2}$ and $\log_{5} n = \frac{\log_{x} n}{\log_{x} 5}$ respectively, 
+we can find that when these values are plugged into $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$ for f(n) c becomes a scaling value between the two fractions:
+
+$c \cdot \frac{\log_{x} n}{\log_{x} 2} = \frac{\log_{x} n}{\log_{x} 5}$
+
+or 
+
+$c \cdot \frac{\log_{x} n}{\log_{x} 5} = \frac{\log_{x} n}{\log_{x} 2}$
+
+in which case c can be utilized to modify either side by a constant value in order to achieve equivalance (for the example $c = \frac{\log_{x} 2}{\log_{x} 5}$ and $c = \frac{\log_{x} 5}{\log_{x} 2}$ respectively)
+AND the base of a log can never be negative (therefore making the value of c to convert always greater than zero), thus the difference in bases does not affect the asymptotic complexity as they can always be changed by some value of c.
```

Comment:
But why does a different constant not change the asymptotic complexity?

Reply:
Because as n gets closer to infinity (and whatever it is plugged into approaches its asymptote) constant values become negligible in comparison to whatever n is plugged into.

Reply:
Good point, changed my examples.